website/docs/r/vpc_endpoint_service.html.markdown

```diff
@@ -54,15 +54,15 @@ The following arguments are supported:
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - The ID of the VPC endpoint service.
-* `availability_zones` - The Availability Zones in which the service is available.
-* `arn` - The Amazon Resource Name (ARN) of the VPC endpoint service.
-* `base_endpoint_dns_names` - The DNS names for the service.
-* `manages_vpc_endpoints` - Whether or not the service manages its VPC endpoints - `true` or `false`.
-* `private_dns_name` - The private DNS name for the service.
-* `service_name` - The service name.
-* `service_type` - The service type, `Gateway` or `Interface`.
-* `state` - The state of the VPC endpoint service.
+* `id` - string - The ID of the VPC endpoint service.
+* `availability_zones` - list(string) - The Availability Zones in which the service is available.
```

Comment:
Looks like availability_zones is actually set(string)

Reply:
Ops, sorry. I would fix it.

https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/data_source_aws_vpc_endpoint_service.go#L30